src/dsn.layer2.thrift

```diff
@@ -75,4 +81,5 @@ struct app_info
     // New fields for bulk load
     // Whether this app is executing bulk load
     14:optional bool    is_bulk_loading = false;
+    15:optional duplication_options dup_options;
```

Comment:
According to the current `trigger_follower_duplicate_checkpoint` implementation. Follower table will store master meta address in app_envs. `duplication_options` store master cluster name and master meta address. Is `duplication_options` necessary? I think you can adding those informations once.

Reply:
`app_envs` is string format which has a little complex to convert vector<rpc_dress>, and the `env `actually just used for show flag info

Reply:
I have difference opinion with you. Structure `app_info` will store in remote storage(zk) and replica server disk, I think we shouldn't add too many fileds in this structure, especially for duplicated structure. 
I still suggest that you can remain only one structure. If the env is only used for showing flag info, it's better to showing the duplication_options but not adding it into envs.

Reply:
`duplication_options` is optional, it will not storage into zk and it also no need.

Add new structure will make code more simple, from `127.0.0.1:34801` string to `rpc_address`(I ever try, we first split , then convert string to int, and then create `rpc_address`) is complex, we don't have to add these redundant code.

Reply:
OK. If `duplication_options will not storage into zk and it also no need`, why would you add it into `app_info`? How about adding `duplication_options` in `create_app_request`?

Reply:
`create_app_request` has existed variable `create_options`, `duplication_options` is a subset, so I put directly in it

Reply:
And, I see， follower will need the info via `app_info`, if put `create_app_request`, I have to create new variable in `meta` and sync to `replica` server, it's very complex

Reply:
Your answer doesn't persuade me. 
`app_info` is used to store app metadata in remote storage and transfer it to replica server. As `duplication_options` is only used in rpc, why not adding it into rpc request?

Reply:
Ok，I change it into `create_app_request`